libnetwork/cnmallocator/manager.go

```diff
@@ -41,7 +43,7 @@ func (d *manager) DeleteNetwork(nid string) error {
 	return types.NotImplementedErrorf("not implemented")
 }
 
-func (d *manager) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo, epOptions map[string]interface{}) error {
+func (d *manager) CreateEndpoint(_ context.Context, nid, eid string, ifInfo driverapi.InterfaceInfo, epOptions map[string]interface{}) error {
 	return types.NotImplementedErrorf("not implemented")
 }
```

Comment:
Unrelated to this PR; are these kind of stubs the ones that go away with your other PR? (did we merge that one yet?)

Reply:
Yeah, my `EndpointDriver` is aiming at making this sort of changes easier and nope, we didn't merge it. And I've another wip branch changing the signature of endpoint-related methods, gonna be fun to rebase everything 😅